.pytool/CISettings.py

```diff
@@ -173,8 +173,6 @@ def GetRequiredSubmodules(self):
         If no RequiredSubmodules return an empty iterable
         '''
         rs = []
-        rs.append(RequiredSubmodule(
-            "CryptoPkg/Library/OpensslLib/openssl", False))

```

Comment:
I think this can be squashed into the first commit.

Reply:
DOne.